src/core/document.js

```diff
@@ -545,16 +545,15 @@ var PDFDocument = (function PDFDocumentClosure() {
       }
       if (isDict(infoDict)) {
         // Only fill the document info with valid entries from the spec.
```

Comment:
This patch is very clearly breaking the assumption listed above, which exists to prevent errors since not all data structures (one example is `Dict`) can be sent from the worker to the main thread.

Assuming first of all that this is even desired functionality, the patch unfortunately cannot be accepted in its current form (furthermore there's also the issue of no tests being include here).

Reply:
Okay, then *what* is the intended functionality? Info dict is clearly in the specs since every PDF accepts custom info dict entries. I do not see any discussions among the issues about this matter.